frontend/config/activityCheckers.ts

```diff
@@ -28,31 +28,36 @@ export const GNOSIS_ACTIVITY_CHECKERS: ActivityCheckers = {
     '0x7Ec96996Cd146B91779f01419db42E67463817a0',
     REQUESTER_ACTIVITY_CHECKER_ABI,
   ),
-};
-
-export const OPTIMISM_ACTIVITY_CHECKERS: ActivityCheckers = {};
+} as const;
 
-export const BASE_ACTIVITY_CHECKERS: ActivityCheckers = {
+const BASE_ACTIVITY_CHECKERS: ActivityCheckers = {
   [ActivityCheckerType.MemeActivityChecker]: new MulticallContract(
     '0x026AB1c5ea14E61f67d245685D9561c0c2Cb39Ba',
     MEME_ACTIVITY_CHECKER_ABI,
   ),
-};
+} as const;
 
-export const MODE_ACTIVITY_CHECKERS: ActivityCheckers = {
+const MODE_ACTIVITY_CHECKERS: ActivityCheckers = {
   [ActivityCheckerType.Staking]: new MulticallContract(
     '0x07bc3C23DbebEfBF866Ca7dD9fAA3b7356116164',
     STAKING_ACTIVITY_CHECKER_ABI,
   ),
-};
+} as const;
+
+const CELO_ACTIVITY_CHECKERS: ActivityCheckers = {
+  [ActivityCheckerType.MemeActivityChecker]: new MulticallContract(
+    '0x3FD8C757dE190bcc82cF69Df3Cd9Ab15bCec1426', // TODO: celo
```

Comment:
this is correct address, you can remove todo  (see activity checker here https://govern.olas.network/contracts/0x00000000000000000000000095d12d193d466237bc1e92a1a7756e4264f574ab)